spring-boot-project/spring-boot-autoconfigure/src/main/resources/META-INF/additional-spring-configuration-metadata.json

```diff
@@ -1677,6 +1677,18 @@
         "replacement": "spring.reactor.debug-agent.enabled"
       }
     },
+    {
+      "name": "spring.redis.lettuce.pool.enabled",
+      "type": "java.lang.Boolean",
+      "description": "Whether to enable lettuce pool. If true, make sure \"commons-pool2\" exists on the classpath.",
+      "defaultValue": true
```

Comment:
There is no default value. It is enabled if `commons-pool2` is on the classpath.